R/population.R

```diff
@@ -49,5 +49,14 @@ Population <- R6::R6Class(
       private$printLine("Number of Individuals", self$count)
       invisible(self)
     }
+  ),
+  private = list(
+    getPathFrom = function(parameterOrPath) {

```

Comment:
@PavelBal I think this follows the philosophy that we implemented so far. Use the ref object or the string